### ISSUE TITLE
Method inside the Job class is not callable

### ISSUE THREAD

**Issue by LeCoupa** [Rocketseat/adonis-bull#71]
The task below will trigger the following error when called.

> TypeError: this.myMethod is not a function

```javascript
export default class MyTask implements JobContract {
  public key = 'theTaskKey'

  public async handle(job) {
    this.myMethod()
  }

  public async myMethod() {}
}
```

Is there a way to call a method from the same class inside `handle()`?

**Comment by HigoRibeiro** [comment]
Hi @LeCoupa !
Unfortunately, as it is being done today it is not possible.
But I will made a PR that fixes this.

**Comment by LeCoupa** [comment]
Thank you 😊

**Comment by jpedroschmitz** [comment]
Hey, I've just published `1.0.0-1` that should fix the issue.

Let me know if you have any problem @LeCoupa 💜

**Comment by LeCoupa** [comment]
Thank you 🙏
